src/Native/System.Native/pal_networking.h

```diff
@@ -90,6 +96,21 @@ struct IPPacketInformation
     int32_t Padding;        // Pad out to 8-byte alignment
 };
 
+struct IPv4MulticastOption
+{
+    uint32_t MulticastAddress; // Multicast address
+    uint32_t LocalAddress;     // Local address
+    int32_t InterfaceIndex;    // Interface index
+    int32_t Padding;           // Pad out to 8-byte alignment
+};
+
+struct IPv6MulticastOption
+{
+    IPAddress Address;      // Multicast address
+    int32_t InterfaceIndex; // Interface index
+    int32_t Padding;        // Pad out to 8-byte alignment
```

Comment:
Rather than add the padding here, which is specific per-compilation-platform, can you pragma ignore the padding warning?